packages/core/src/domain/telemetry/telemetry.spec.ts

```diff
@@ -31,6 +42,39 @@ describe('telemetry', () => {
     expect(notifySpy).toHaveBeenCalledTimes(1)
   })
 
+  describe('addTelemetryConfiguration', () => {
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏it could be nice to add a test to ensure that `telemetryConfigurationEnabled` depends on `telemetryEnabled`

Reply:
https://github.com/DataDog/browser-sdk/blob/1efc794192f0c3d71d3ece214e07869728f23ed3/packages/core/src/domain/telemetry/telemetry.spec.ts#L77